src/widgets/JSONForm/templates/Field.js

```diff
@@ -59,7 +59,7 @@ export const Field = ({
   );
 
   if (schemaIsRoot(id)) {
-    return (
+    return label ? (
```

Comment:
Should this be `!label`?

Reply:
The idea (and maybe it is hacky) was to not display the paper wrapper with header if there isn't a title present in the JSONSchema 😅  (for when the `JSONFrom` is loaded inside a component with its own title)